prisma/schema.prisma

```diff
@@ -24,3 +24,23 @@ model Password {
   user   User   @relation(fields: [userId], references: [id])
   hash   String
 }
+
+// Originally thought it was going to be a many-to-many relationship
+// Thought about it again and decided it was one-to-many
+// Andre - looking forward to your feedback!
+model Group {
```

Comment:
Since we already have the `User` model - how do you think the user object is associated with groups and events? (We can also discuss this next session, so there is no rush to implement this in this PR).

Reply:
Okay, yes if we can discuss this at our meeting today, that would be most helpful for me!